Roizor.md

```diff
@@ -0,0 +1,39 @@
+---
+name: "@Roizor"
+project: "Blue Board"
+---
+
+# Blue Board
+
+## Summary
+
+  I am going to build a 65% mechanical keyboard but use custom firmware (a fork of QMK). The keyboard will be used for everything I do (home, school). 
+  I'll also create a custom key layout for the keyboard.
+  I'm excited to build this because it's always been my dream to build a mechanical keyboard and modify it's firmware at the low level. It'll be my first time building a keyboard as well.
+  I've also never truly explored C programming and I feel this project can help introduce me to it. As for what I am going to do for the firmware, my laptop builtin keys have special functions, such as trackpad on/off, key lights on/off, and a number pad. I want to include these settings on the keyboard itself because I think it will challenge me and be a fun first experience (for the most part). 
+
+## Plan
+
+From days 1-3, I will be assembling the keyboard with regular tests to make sure I haven't broken anything yet.
+After some research and playing with C on day 4 to get the hang of it, I will start to develop the firmware itself.
+On days 4-8, I will be continuing to program the board's firmware with regular tests & debugging sessions.
+On day 9, I'll do in depth testing to make sure everything is correct in case something was missed.
+On the final day (10), because everything will be done I will customize the keyboard and have 1 final look. 
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Lube for switches (Krytox 103 + brush)  | https://1upkeyboards.com/shop/parts-and-tools/lubricant/switch-lubricant-3ml | $11.00  |
+| NK65 PCB/board (includes most things i will need) | https://novelkeys.com/collections/keyboards/products/nk65-entry-edition?variant=36839795589287  | $95.00 |
+| Durock Linear Switches| https://divinikey.com/products/durock-linear-switches?variant=39872545226817 | $38.88|
+| Keycaps (Artifact Bloom series Black on White) | https://drop.com/buy/artifact-bloom-series-keycap-set-black-on-white| $35.00|
+| Total           |                                       | $179.88 |
+
+## Alternative/backup items
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| DZ65 PCB/board | https://divinikey.com/collections/keyboards/products/kbdfans-dz65-rgb-hot-swap-v2-pcb?variant=39278196883521  | $95.00 |
+| Cream Dream Linear Switches| https://kbdfans.com/collections/switches/products/cream-dream-linear-switches | $68.88 |
+| Keycaps (Keyreative ABS Blank Cherry Keycaps) |https://thekey.company/collections/keycaps/products/keyreative-abs-blank-cherry-keycaps?variant=31157483307097 | $35.00|
+| Durok V2 Stabilizers | https://prevailkeyco.com/collections/stabilizers/products/durock-v2-stabilizers?variant=42451809370366 | $4.00 |
```

Comment:
Correct me if I'm wrong, but shouldn't stabilizers be necessary to your build and not a backup/alternative? I don't see them in the table above.

Reply:
The NK65 board comes with them

Reply:
Oh. My bad. the NK65 does not come with stabilizers. Sorry, i'll move that up.